virt-who.8

```diff
@@ -50,13 +50,13 @@ Choose virtualization backend that should be used to gather host/guest associati
 Choose where the host/guest associations should be reported
 .TP
 \fB\-\-sam\fR
-[Deprecated] Report host/guest associations to the Subscription Asset Manager or Satellite 6 [default]
+[Deprecated] Report host/guest associations to the Subscription Asset Manager, Satellite 6, or Stage Candlepin. This option specifies the default behaviour, and thus it is not used [default]
```

Comment:
'Stage Candlepin' is not what the end user should see. Let's use 'Red Hat Subscription Management (RHSM)'

Reply:
Updated